crypto/mod.ts

```diff
@@ -38,6 +39,7 @@
  *
  * Wasm/Rust
```

Comment:
this would turn into being at the bottom of the example above; these titles should come after their corresponding example tags

Reply:
Fixed